pkg/pstore/api.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package pstore // import "storj.io/storj/pkg/pstore"
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path"
+)
+
+type argError struct {
+	arg string
+	msg string
+}
+
+func (e *argError) Error() string {
+	return fmt.Sprintf("HashError (%s): %s", string(e.arg), e.msg)
+}
+
+type fsError struct {
+	path string
+	msg  string
+}
+
+func (e *fsError) Error() string {
+	return fmt.Sprintf("FSError (%s): %s", e.path, e.msg)
+}
+
+func Store(hash string, r *bufio.Reader, dir string) error {
+	fmt.Println("Storing...")
```

Comment:
make sure to use log, instead of fmt.Print, since that goes to stdout, which is also sometimes where your file goes.